examples/with-tailwindcss/package.json

```
@@ -3,21 +3,23 @@
   "scripts": {
     "dev": "next dev",
     "build": "next build",
-    "start": "next start"
+    "start": "next start",
+    "lint": "next lint"
   },
   "dependencies": {
-    "next": "latest",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2"
+    "next": "12.1.6",
```

This should not be pinned. Let's keep `latest`:

```suggestion
    "next": "latest",
```